external/deep-object-reid/torchreid_tasks/utils.py

```diff
@@ -400,6 +400,41 @@ def on_test_batch_end(self, batch=None, logs=None):
         self.update_progress_callback(self.get_progress())
 
 
+class OptimizationProgressCallback(TimeMonitorCallback):
+    """ Progress callback used for optimization using NNCF
+        There are four stages to the progress bar:
+           - 5 % model is loaded
+           - 10 % compressed model is initialized
+           - 10-100 % compressed model is being fine-tuned
+    """
+    def __init__(self, update_progress_callback: UpdateProgressCallback, load_progress: int = 5,
+                 initialization_progress: int = 5, **kwargs):
```

Comment:
To be honest, the names `load_progress` and `initialization_progress` confuse me. As far as I understand, these are the progress bar step ranges. I would recommend to think on more clear names, for example, `loading_step_progress_range` and `initialization_step_progress_range` or use min/max values to specify progress ranges.

Reply:
How about `loading_step_progress_percentage` and `initialization_step_progress_percentage`? `range` for me is a a little bit confusing since it usually involves two values, but here it means by how much percent we increase the progress bar after the corresponding step is finished